wb_modbus_device_editor/template_manager.py

```diff
@@ -104,14 +100,14 @@ def translate(self, title, language="ru"):
 
 
 class TemplateManager:
-    _DEFAULT_TEMPLATES_DIR = "templates"
-    _VERSION_FILENAME = "version"
+    _DEFAULT_TEMPLATES_DIR = os.path.join(pathlib.Path.home(),"templates")
```

Comment:
```suggestion
    _DEFAULT_TEMPLATES_DIR = os.path.join(pathlib.Path.home(),".wb-templates")
```

а то потом люди будут гадать, что за темплейты

Reply:
поправлено